Dirty patch that makes ext3 work again on 3.5 and 3.6 kernels,
on mips n32.

http://www.linux-mips.org/archives/linux-mips/2012-11/msg00030.html

diff --git a/fs/ext3/dir.c b/fs/ext3/dir.c
index 92490e9..bf63d7b 100644
--- a/fs/ext3/dir.c
+++ b/fs/ext3/dir.c
@@ -228,6 +228,7 @@ out:
 
 static inline int is_32bit_api(void)
 {
+	return 1;
 #ifdef CONFIG_COMPAT
 	return is_compat_task();
 #else
